Cython/Utility/ModuleSetupCode.c

```diff
@@ -641,12 +649,19 @@ class __Pyx_FakeReference {
 #else
   #define __Pyx_BUILTIN_MODULE_NAME "builtins"
   #define __Pyx_DefaultClassType PyType_Type
-#if PY_VERSION_HEX >= 0x030B00A1
+#if CYTHON_COMPILING_IN_LIMITED_API
+    static CYTHON_INLINE PyObject* __Pyx__PyCode_New() {
+        // Don't know how to do this reliably right now... improve!
+        Py_RETURN_NONE;
+    }
+    // Defining a macro ignores the args so we don't have to define CO_OPTIMIZED etc. in limited API
+    #define __Pyx_PyCode_New(...) __Pyx__PyCode_New()
```

Comment:
Weren't macros with `...` arguments an issue in C89? I faintly recall that this was a gcc extension at the time.